config/language/msg_template.txt

```diff
@@ -25,8 +25,10 @@ Artwork Label;Artwork Label
 Artwork Name;Artwork Name
 Artwork Path;Artwork Path
 Attract-Mode detected emulator(s) that can be imported automatically.  Import them now?;Attract-Mode detected emulator(s) that can be imported automatically.  Import them now?
-Auto-detect emulators?;Auto-detect emulators?
+Auto-detect Emulators;Auto-detect Emulators
```

Comment:
I'm cleaning up the translation files now. Where did you find this line in the code?